ArduinoCode/Driver-IO/BasicRelay/BasicRelay.ino

```diff
@@ -0,0 +1,50 @@
+#include <mcp_can.h>
+#include <mcp_can_dfs.h>
+#include <SPI.h>
+
+const int RELAY_PIN = 9;
+
+const int CAN_ID = 0x03; // device's CAN id
+const int SPI_CS_PIN = 10; // base CAN pin
+
+unsigned char len = 0; // length of incoming message
+unsigned char buf[1]; // memory for incoming message
+unsigned long canId; // CAN id of incoming message
+
+MCP_CAN CAN(SPI_CS_PIN); // setup can device
+
+/**
+ * Start CAN and intialize digital pin
+ */
+void setup() {
+  Serial.begin(115200); // baud rate for CAN
```

Comment:
Isn't this the baud rate for the serial output and nothing to do with CAN?

Reply:
Just wanna make sure the comment is right.

Reply:
Yes good catch. Baud rate is CAN.begin(CAN_500KBPS, ....